app/src/org/commcare/tasks/ZipTask.java

```diff
@@ -200,21 +72,24 @@ private boolean zipTargetFolder(File targetFilePath, String zipFile) throws IOEx
         return false;
     }
 
-    private boolean zipFolder(File[] files, String zipFile, ZipOutputStream zos) throws IOException {
-        Log.d(TAG, "827 zipping folder with files: " + files[0] + ", zipFile: " + zipFile);
+    private void zipInstanceFolder(File[] toBeZippedFiles, String zipFilePath, ZipOutputStream zos)

```

Comment:
The `zipFilePath` argument is only used in a log statement, which feels a bit strange. Can you move the log statement outside of this method call and remove the `zipFilePath` arg?